### PR TITLE
🐛 PercyDOM.serialize returns fix now #172

### PR DESCRIPTION
**Pull request by samarsault**
Fixes https://github.com/percy/percy-selenium-java/issues/172

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,3 +124,22 @@ If you have a previous Percy configuration file, migrate it to the newest versio
...
+Install maven:
+
+```sh-session
+$ brew install mvn
+```
+
+Run tests:
+
+```
+npx percy exec -- mvn test
```
Comment (itsjwala) [inline README.md:144]:
```suggestion
npm run test
```

add test script in package.json 

```
npx percy exec --testing -- mvn test
```

File: README.md [hunk 2]
```diff
@@ -124,3 +124,22 @@ If you have a previous Percy configuration file, migrate it to the newest versio
...
+Install/update `@percy/cli` dev dependency (requires Node 14+):
+
+```sh-session
+$ npm install --save-dev @percy/cli
+```
+
+Install maven:
+
+```sh-session
+$ brew install mvn
```
Comment (itsjwala) [inline README.md:138]:
```suggestion
$ brew install maven
```

is it maven?

Comment (samarsault) [reply README.md:138]: Both are same

File: README.md (hunk 1 above)
Comment (samarsault) [reply README.md:144]: It's not working that way due to some reason

File: README.md [hunk 3]
```diff
@@ -124,3 +124,22 @@ If you have a previous Percy configuration file, migrate it to the newest versio
...
+```
+
+Run tests:
+
+```
+npm test
```
Comment (itsjwala) [inline README.md:144]:
```suggestion
npm run test
```

?

Comment (samarsault) [reply README.md:144]: Both work

**Pull request merged**